### PR TITLE
Cpg Integration

### PR DESCRIPTION
**Pull request by michaelyoukeim**
Integrated the sootup.codepropertygraph module.

### CODE REVIEW

File: sootup.codepropertygraph/src/test/java/sootup/codepropertygraph/CpgTestSuiteBase.java [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    File folder = file.getParentFile();
+    if (folder != null && !folder.exists()) {
+      folder.mkdirs();
+    }
+
+    try (BufferedWriter writer = new BufferedWriter(new FileWriter(file))) {
+      writer.write(dotGraph);
+    } catch (IOException e) {
+      throw new RuntimeException(e);
+    }
```
Comment (swissiety) [inline sootup.codepropertygraph/src/test/java/sootup/codepropertygraph/CpgTestSuiteBase.java]: can we do that in memory?

**Review by swissiety — COMMENTED** [review]
lets add a page in docs/ to documenting its existance and inform about a usage examples/entrypoints to developers how its intended to be used

File: sootup.codepropertygraph/src/test/java/sootup/codepropertygraph/CpgTestSuiteBase.java (hunk 1 above)
Comment (michaelyoukeim) [reply sootup.codepropertygraph/src/test/java/sootup/codepropertygraph/CpgTestSuiteBase.java]: Done

**Comment by michaelyoukeim** [comment]
@swissiety @stschott The user documentation will be added shortly.

**Review by stschott — APPROVED** [review]

**Pull request merged**